src/main/java/dev/andrylat/vomelianchuk/card_number_operations/consoleutils/UserConsoleIO.java

```diff
@@ -2,20 +2,29 @@
 
 import dev.andrylat.vomelianchuk.card_number_operations.enums.PaymentSystem;
 import dev.andrylat.vomelianchuk.card_number_operations.exceptions.CardValidationException;
+import dev.andrylat.vomelianchuk.card_number_operations.exceptions.DataMortgageCalculatorException;
 import dev.andrylat.vomelianchuk.card_number_operations.exceptions.WrongPaymentSystemException;
 
 import java.io.InputStream;
 import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.InputMismatchException;
 import java.util.List;
 import java.util.Scanner;
 
 public class UserConsoleIO {
```

Comment:
This class is responsible for multiple logics. If your application has 5-10, here you will have 5-10x more responsibilities?

Reply:
Fixed here: 5137270